Neos.ContentRepository.Core/Classes/Feature/WorkspaceCommandHandler.php

```diff
@@ -432,6 +434,35 @@ static function ($handle) use ($rebaseableCommands): void {
         yield $this->removeContentStreamWithoutConstraintChecks($workspace->currentContentStreamId);
     }
 
+    /**
+     * This layer can be removed if there are no legacy events expected during rebasing
+     * @return iterable<EventEnvelope>
+     */
+    private function triggerConflictForLegacyEvents(EventStreamInterface $eventStream, bool $dropLegacyEvents): iterable
```

Comment:
this might not be soo clever after all because this enforced to use of `force` and later in the code during the forced rebase other changes which were not communicated might also be dropped.

So maybe its just best to silently ignore any `CopyNodesRecursively` and basically auto drop them for publish, discard and rebase?

We have said for 3 betas that this will happen ...